dlrover/python/master/node/worker.py

```diff
@@ -285,7 +286,16 @@ def has_exited_worker(self):
             if (
                 worker.exit_reason == NodeExitReason.FATAL_ERROR
                 or worker.status == NodeStatus.SUCCEEDED
+                or (
+                    worker.status == NodeStatus.DELETED
+                    and worker.reported_status[0]
```

Comment:
Better to add a method under the `Node` class that only exposes the reported status(status without timestamp), to avoid the widespread use of `tuple[magic number]`.